recipes/mitos/meta.yaml

```diff
@@ -39,8 +35,7 @@ test:
     - mitos
     - mitos.bedfile
     - mitos.blast
-    # does not work wo reportlab (see above)
-    # - mitos.draw
+    - mitos.draw
```

Comment:
Comment state this does not work without reportlab which is no longer used, just checking if this is on purpose

Reply:
Jep. Noticed, that reportlab comes (now) with biopython.

Reply:
Thanks for checking.